Tests/TuistAutomationTests/Utilities/TargetRunnerTests.swift

```diff
@@ -18,7 +18,7 @@ final class TargetRunnerErrorTests: XCTestCase {
         )
         XCTAssertEqual(
             TargetRunnerError.runningNotSupported(target: .test(platform: .iOS, product: .app)).description,
-            "Cannot run Target - the platform iOS and product type app are not currently supported."
+            "Product type app of Target is not runnable"
```

Comment:
Nitpick: I'd add "of **target** Target" so that the reader of the error knows that we are referring to the target. What are the runnable targets? What if we include them here?